handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -174,12 +174,11 @@ func getClaim(t *jwt.Token, claim string) (interface{}, bool) {
 	if claims, ok := t.Claims.(jwt.MapClaims); ok {
 		c, has := claims[claim]
 		if !has {
-			log.Printf("Missing claim %q for token %v", claim, maskToken(t.Raw))
 			return "", false
 		}
 		return c, true
 	}
-	log.Printf("Missing claim %q for token", claim)
+	log.Printf("Missing claim %q for token %v", claim, maskToken(t.Raw))	
```

Comment:
not that much of a "if you say so". The function returns an boolean indicating if the claim is there or not. Clients can decide to log the message if necessary. Thanks for the PR!